docs/integrations/common-workflow-steps/build-cache/cache-pull.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Cache Pull 
+description: Enhance your development process with Cache Pull to quickly retrieve and reuse stored data, boosting efficiency and performance.
+tags: [cache pull, efficiency, dependencies, cache structure]
+sidebar_position: 2
+---
+
+import Screenshot from '@site/src/components/Screenshot';
+
+# Cache Pull
+
+[**Cache Push**](./cache-push) uploads the cache archive file to a remote location, as we explained in detail in the **Cache Push** step. On the other hand, **Cache Pull** downloads and extracts that archive file in the build pipeline, restoring all files and folders to their original locations.
+
+:::warning
+[**Cache Push**](./cache-push) and **Cache Pull** components should work in coordination on the same cache file. Therefore, to download the pushed cache, the **Cache Pull** must have the same cache label as the **Cache Push**.
+:::
+
+:::info
+If you need to use the cached folder in a different branch or a separate project, you have the capability to modify the values of `$AC_GIT_BRANCH` or `$AC_BUILD_PROFILE_ID`. For further information, please check out the following documentation:
+- [How to Share Files Between Pipelines](./how-to-share-file-between-pipelines)
```

Comment:
Links should be the exact path.

Wrong:
`(./cache-push)`

Correct:
`(/workflows/common-workflow-steps/cache-push)`